app/build.gradle

```diff
@@ -20,6 +20,8 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Looks like this was added automatically by AS on project init. Seems it's just supposed to provide out of the box jar compiling, as you probably guessed and thus wondered why it was necessary. It's not, so good call :D